acl/acl-lib.pl

```diff
@@ -1693,7 +1693,7 @@ sub encrypt_password
 	# Try detecting system default first
 	if (&foreign_available('useradmin')) {
 		&foreign_require('useradmin');
-		return &useradmin::encrypt_password($pass, $salt);
+		return &useradmin::encrypt_password($pass, $salt, 'force_system_detection');
```

Comment:
Because, I have these magic numbers. You can never know what's happening, unless looking at the sub desription or code itself.

I could change it to `1` for you if you find it cleaner.